openreview/conference/invitation.py

```diff
@@ -901,7 +901,7 @@ def __init__(self, conference):
             multiReply = False,
             reply = {
                 'writers': {
-                    'values-regex': [conference.get_program_chairs_id()],
+                    'values-regex': conference.get_program_chairs_id(),
```

Comment:
good catch!

Given the string is not regex and it is valid group id, let's change the field type to  `values`, like signatures.

Reply:
Done!